certbot/certbot/compat/misc.py

```diff
@@ -112,7 +112,8 @@ def underscores_for_unsupported_characters_in_path(path: str) -> str:
     return drive + tail.replace(':', '_')
 
 
-def execute_command(cmd_name: str, shell_cmd: str, env: Optional[dict] = None) -> Tuple[str, str]:
+def execute_command_status(cmd_name: str, shell_cmd: str,
+                           env: Optional[dict] = None) -> Tuple[int, str, str]:
     """
     Run a command:
```

Comment:
nit: can we add a comment like "this function differs from `execute_command` in that it returns the status code, and does no logging internally"?